### ISSUE TITLE
About training from scratch

### ISSUE THREAD

**Issue by zanchangtong** [siavash-khodadadeh/UMTRA-Release#1]
Thank you for providing the code of the paper. I have some problems after reading the paper and recurring some experiments.
What is the setting for training from scratch? Does the number of iterations of the inner loop have to be set to 1? 

All hyperparameters are set as follows:
dataset=omniglot 
epoch=30000 --epoch number
img_size=28
k_qry=4  --k shot for query set
k_spt=1  --k shot for support set
meta_lr=0.001  --meta-level outer learning rate
n_way=5  
batch_size=32  --meta batch size
update_lr=0.05  --task-level inner update learning rate
update_step=5  --task-level inner update steps
update_step_test=50  --update steps for finetunning
The random pixel zeroing algorithm is the same as that provided in this project.

And, when I conduct training from scratch or experiment with random pixel zeroing, the accuracy of all 5-way 1-shot tasks is below 40%.
